Azordev/did-app
# Refactor/removing unneeded code

## Description

I refactored some components and pages and added documentation in Readme.MD

## Summary

- Removed handleListQuery function since it was not dev-friendly and is not scalable
- Refactor all pages and components to the new VueJs 3 script setup sintaxis
- Removed unused code, as the pagination and search in pages where it is not used anymore, as Events and Providers
- Fixed all the TypeScript errors on the project
- Added recommended extensions

## How to test

### Dev test:

Try running the project and testing that all features work

### QA test:

Enter using the development links and try to check if all features works properly
